src/h3lib/include/polygonAlgos.h

```diff
@@ -53,7 +53,7 @@
 
 /** Macro: Normalize longitude, dealing with transmeridian arcs */
 #define NORMALIZE_LON(lon, isTransmeridian) \
-    (isTransmeridian && lon < 0 ? lon + (double) M_2PI : lon)
+    (isTransmeridian && lon < 0 ? lon + (double)M_2PI : lon)
```

Comment:
This change shouldn't have been possible - the error is this file is not in `LIB_SOURCE_FILES` in CMakeLists.txt

Reply:
Oh OK I'll wait on that.